components/patient-data/ui/src/main/resources/PhenoTips/UIX_Field__genes.xml

```diff
@@ -137,10 +137,12 @@
         var infoTool = new Element('span', {'class' : 'fa fa-info-circle xHelpButton gene-info', 'title' : geneNames[i]});
         var icon_helpController = new PhenoTips.widgets.HelpButton(infoTool);
         var triangle = new Element('span', {'class': "fa fa-exclamation-triangle", title: '$services.localization.render('phenotips.patientSheetCode.termSuggest.nonStandardGenesymbol')'}).update(' ');
-        if (isSuggestSelected)
+        if (isSuggestSelected) {
```

Comment:
yes thats good but its not what I mean. Also is the variable ```isSuggestSelected``` still appropriately named. Would something like ```isViewMode``` be better?